### PR TITLE
Update Simplified Chinese localization

### PR DESCRIPTION
**Pull request by taozhiyu**
Update Simplified Chinese localization

- updated documentation
- More fluency

fix issue in [this commit](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d52402196e295c82ca73c8194d18a57c6d02a02b#r95588217)
In addition to the localization of the readme file, this pr has (just) changed the type of types/tampermonkey
I'm not sure if I should wait for this pr to be merged before modifying...

If anyone knows can give me a heads up 😆 

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/tampermonkey/tampermonkey-tests.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@ if (window.onurlchange === null) {
     window.addEventListener('urlchange', ({ url }) => console.log(url));
     // or
     window.addEventListener('urlchange', info => console.log(info.url));
+
+    // window.addEventListener('click', (event) => console.log(e.url));
```
Comment (nikolay-borzov) [inline types/tampermonkey/tampermonkey-tests.ts:12]: Why do we need this?

Comment (taozhiyu) [reply types/tampermonkey/tampermonkey-tests.ts:12]: As in the comment on the next line, the comment here is meant to say that this is incorrect and can be identified by types, as only urlchange events return parameters with url objects, see commit [here](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d52402196e295c82ca73c8194d18a57c6d02a02b#r95588217) for details, perhaps the unnecessary comment here could be removed?

**Comment by rbuckton** [comment]
> In addition to the localization of the readme file, this pr has (just) changed the type of types/tampermonkey
> I'm not sure if I should wait for this pr to be merged before modifying...

I think it would be better to move the tampermonkey changes to a separate PR, to keep this focused on the localization changes as is mentioned in the title.

**Comment by taozhiyu** [comment]
> > In addition to the localization of the readme file, this pr has (just) changed the type of types/tampermonkey
> > I'm not sure if I should wait for this pr to be merged before modifying...
> 
> I think it would be better to move the tampermonkey changes to a separate PR, to keep this focused on the localization changes as is mentioned in the title.

Yes I agree.

Split into #63899 and #63900.

Thanks.

**Pull request closed without merging**